packages/react/src/components/FormattingToolbar/DefaultButtons/BasicTextStyleButton.tsx

```diff
@@ -101,7 +101,7 @@ export const BasicTextStyleButton = <Style extends BasicTextStyle>(props: {
     return !!selectedBlocks.find((block) => block.content !== undefined);
   }, [basicTextStyleInSchema, selectedBlocks]);
 
-  if (!show) {
+  if (!show || !editor.isEditable) {
```

Comment:
why do we move the editable checks here? Doesn't it make more sense to make sure the entire formattingtoolbar is not shown?

Reply:
Unfortunately not for the tiny edge case of being able to download files while the editor is uneditable, which is done via the formatting toolbar

Reply:
Hmm, wouldn't it be cleaner then to make sure clicking the button downloads files in non-editable state? That saves a lot of logic in the buttons, as we can then just disable the formatting toolbar when non editable.

I also think that's semantically better, as for a "formatting toolbar" I'd expect I'd only be able to use it when I can actually edit / format the text. 

wdyt?

Reply:
Also r.e. semantics I agree, but I think the root issue of that is really having the download button in the formatting toolbar to begin with. I reckon at some point we will probably change to a file-specific menu for file blocks like Notion has.

Reply:
Ok. I still don't really like it, but agree there's not a better solution without significantly more effort. Let's go for this option for now. Can you add the reasoning (that and why we allow the formatting toolbar to show even when the editor is not editable, and that we instead do a per-button check) as a comment?